documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,63 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo \
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo dnf config-manager --add-repo \
+	https://yum.dockerproject.org/repo/main/fedora/24
+$ sudo dnf install docker-engine-1.12.1-1.fc24
```

Comment:
We need to add the step documented here, before installing `docker-engine`:
https://github.com/01org/cc-oci-runtime/issues/687#issuecomment-283209057

so we do not get this error: 

`Error: Public key for docker-engine-1.12.1-1.fc24.x86_64.rpm is not installed`